roles/pulp_repository/tasks/rpm.yml

```diff
@@ -19,8 +19,8 @@
     validate_certs: "{{ pulp_validate_certs | bool }}"
     name: "{{ pulp_repository_rpm_repos[repository_index].name }}-remote"
     ca_cert: "{{ pulp_repository_rpm_repos[repository_index].ca_cert | default(omit) }}"
-    client_cert: "{{ pulp_repository_rpm_repos[repository_index].client_cert | default(omit) }}"
-    client_key: "{{ pulp_repository_rpm_repos[repository_index].client_key | default(omit) }}"
+    client_cert: "{{ pulp_repository_rpm_repos[repository_index].client_cert | default('') }}"
+    client_key: "{{ pulp_repository_rpm_repos[repository_index].client_key | default('') }}"
```

Comment:
ah, I see what happened. Still, I'm not sure this is the right place for the fix.

1. It's good to rely on the module defaults
2. It's nice to be able to provide a subset of parameters and leave the rest as 'don't care' when updating existing remotes
3. This is now inconsistent with other parameters.

I suggest we explicitly set these in stackhpc-kayobe-config.

Reply:
Hmm, you mean setting `client_cert` and `client_key` to '' in kayobe-config when switching from cert to password? (To 'force' the role to remove certs from rpm remotes already configured in pulp)

Reply:
Yes

Reply:
I think with the ,,wave'' of upgrades, we'll get more people with the same problem - would be good to at least document it or fix it in s-k-c.

Reply:
ah, I didn't know that @m-bull. I thought you said it "just worked" :D

Reply:
+1 for fix in s-k-c